content/applications/productivity/iot/config/connect.rst

```diff
@@ -1,81 +1,89 @@
-===================================
-Connect an IoT Box to your database
-===================================
+=======================================
+Connect an IoT Box to the Odoo database
+=======================================
 
-Install the Internet of Things (IoT) App on your Odoo Database.
+Install the :menuselection:`Internet of Things (IoT)` App on the Odoo database.
 
 .. image:: connect/connect01.png
    :align: center
+   :alt: The Internet of Things (IoT) App on the Odoo database.
 
-Go in the IoT App and click on Connect on the IoT Boxes page.
+Go to the :menuselection:`IoT` App and click on :guilabel:`Connect`` on the
+:abbr:`IoT (Internet of Things)` IoT Boxes page.
 
-.. image:: connect/connect02.png
+.. image:: connect/connect09.png
    :align: center
+   :alt: Connecting an IoT box to the Odoo database.
 
-Follow the steps to connect your IoT Box.
+Follow the steps to connect the IoT Box.
 
-.. image:: connect/connect03.png
+.. image:: connect/connect-iot-box.png
    :align: center
+   :alt: Connection steps for a wired connection or WiFi connection.
+
+.. note::
+   Ensure the :abbr:`IoT (Internet of Things)` box is flashed with the most up-to-date
+   :abbr:`IoT (Internet of Things)` image.
 
 Ethernet Connection
 ===================
 
-#. Connect to the IoT Box all the devices that have to be connected with
-   cables (ethernet, usb devices, etc.).
+#. Connect all the devices that have to be connected with cables to the
+   :abbr:`IoT (Internet of Things)` box (ethernet, usb devices, etc.).
 
-#. Power on the IoT Box.
+#. Power on the :abbr:`IoT (Internet of Things)` Box.
 
-#. Read the Pairing Code from a screen or a receipt printer connected to the IoT Box.
+#. Read the :guilabel:`Pairing Code` from a screen or a receipt printer connected to the
```

Comment:
If they have no screen attached for some reason, they can click on "customer display" (button on the IoT box home page) to be able to get the pairing code.
Maybe mention in note (or in the trouble shootings) that the pairing code is available for the 5 minutes after the first connection of the IOT. 
If for some reason they have to reconnect it later without having to reflash it, they will have to pass through the token.

Reply:
Noted, I've added this as a note in the troubleshooting section. Thanks for suggesting it!